js/main.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+var PINS_COUNT = 8;
+var PIN_LOCATION_X_MIN = 0;
+var PIN_LOCATION_X_MAX = 1200;
+var PIN_LOCATION_Y_MIN = 130;
+var PIN_LOCATION_Y_MAX = 630;
+
+var map = document.querySelector('.map');
+var pinsGeneratedData = {
+  'prices': [1000, 1200, 1500, 2500, 3000],
+  'types': ['palace', 'flat', 'house', 'bungalo'],
+  'rooms': [1, 2, 3, 4],
+  'guests': [1, 2, 3, 4],
+  'checkin': ['12:00', '13:00', '14:00'],
+  'features': ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'],
+  'photos': ['http://o0.github.io/assets/images/tokyo/hotel1.jpg', 'http://o0.github.io/assets/images/tokyo/hotel2.jpg', 'http://o0.github.io/assets/images/tokyo/hotel3.jpg'],
+};
+
+var getRandomNumber = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+};
+
+var getRandomElement = function (elements) {
+  return elements[Math.floor(Math.random() * (elements.length))];
+};
+
+var getRandomElements = function (elements) {
+  var randomElements = elements.slice();
+  for (var i = randomElements.length - 1; i > 0; i--) {
+    var j = Math.floor(Math.random() * (i + 1));
```

Comment:
Тут нужно брать случайный индекс из всего массива
```suggestion
    var j = Math.floor(Math.random() * randomElements.length);
```